### PR TITLE
Added OlehTysovskyi animation

### PR DESCRIPTION
**Pull request by OlehTysovskyi**

### CODE REVIEW

File: src/services/config-loader.ts [hunk 1]
```diff
@@ -106,7 +108,10 @@ export class MainConfigLoaderService {
         results.push(config);
         counter++;
       } catch (exc) {
-        console.error(`Error occured while processing student '${name}' - ${email}.`, exc);
+        console.error(
+          `Error occured while processing student '${name}' - ${email}.`,
+          exc
+        );
```
Comment (aslobodyanuk) [inline src/services/config-loader.ts:114]: Please do not change this code. Revert this change.

**Comment by OlehTysovskyi** [comment]
Revert console.error

File: src/services/config-loader.ts [hunk 2]
```diff
@@ -40,10 +40,10 @@ import { DmytroSerafymConfigLoaderService } from "./students/dmytro_serafym";
 import { VictoriaKondratskaConfigLoaderService } from "./students/victoria-kondratska";
 import { AndriiKozynyiConfigLoaderService } from "./students/andrii-kozynyi";
 import { MariiaHusakConfigLoaderService } from "./students/mariia-husak";
+import { OlehTysovskyiConfigLoaderService } from "./students/oleh-tysovskyi";
 
 export class MainConfigLoaderService {
   private services: IConfigLoaderService[];
-
```
Comment (aslobodyanuk) [inline src/services/config-loader.ts:46]: Do not remove those lines

File: src/services/config-loader.ts [hunk 3]
```diff
@@ -110,6 +112,7 @@ export class MainConfigLoaderService {
       }
     }
 
+    
     return results;
   }
-}
```
Comment (aslobodyanuk) [inline src/services/config-loader.ts:115]: Still some formatting here...
